src/Core/tests/DeviceTests/Handlers/SearchBar/SearchBarHandlerTests.Android.cs

```diff
@@ -1,16 +1,60 @@
-﻿using Microsoft.Maui.Handlers;
+﻿using System.Linq;
+using System.Threading.Tasks;
+using Android.Widget;
+using Microsoft.Maui.DeviceTests.Stubs;
+using Microsoft.Maui.Handlers;
+using Xunit;
 using SearchView = AndroidX.AppCompat.Widget.SearchView;
 
 namespace Microsoft.Maui.DeviceTests
 {
 	public partial class SearchBarHandlerTests
 	{
+		[Fact(DisplayName = "CharacterSpacing Initializes Correctly")]
+		public async Task CharacterSpacingInitializesCorrectly()
+		{
+			var xplatCharacterSpacing = 4;
+
+			var searchBar = new SearchBarStub()
+			{
+				CharacterSpacing = xplatCharacterSpacing,
+				Text = "Test"
+			};
+
+			float expectedValue = searchBar.CharacterSpacing.ToEm();
+
+			var values = await GetValueAsync(searchBar, (handler) =>
+			{
+				return new
+				{
+					ViewValue = searchBar.CharacterSpacing,
+					NativeViewValue = GetNativeCharacterSpacing(handler)
+				};
+			});
+
+			Assert.Equal(xplatCharacterSpacing, values.ViewValue);
+			Assert.Equal(expectedValue, values.NativeViewValue, 4);
```

Comment:
I had to add rounding in order to get the tests to pass. I picked 4 as that was what the "em" coefficient precision was.

Reply:
As I type this, I feel my own reviewing self say this is probably meant to be a constant instead of a magic number...

Reply:
Yup. Took 3 minutes for me to give in and make a constant.